array-manipulation-2/src/take.js

```diff
@@ -0,0 +1,12 @@
+/* exported take */
+function take(array, count) {
+  var arrayLength = [];
+  for (var i = 0; i <= count - 1; i++) {
+    if (array.length <= 1) {
+      return array;
```

Comment:
It's best to get special cases like this out of the way before the loop starts if you can.